scripts/qa_crowbarsetup.sh

```diff
@@ -2294,6 +2311,16 @@ function hacloud_configure_cluster_members()
                 "['attributes']['pacemaker']['stonith']['per_node']['nodes']['$node']['params']" "''"
         done
     fi
+
+    if [ -n "$want_sbd" ] ; then
+        for node in $@; do
+            sbd_device=$(ssh $node echo '/dev/disk/by-id/scsi-$(lsscsi -i | grep LIO | tr -s " " |cut -d " " -f7)')
```

Comment:
This like is cool and tricky. I would suggest to refactor it out and reuse it in L#2058 too. But not important for me